### ISSUE TITLE
Translation problem for new products in french

### ISSUE THREAD

**Issue by henri9813** [PrestaShop/PrestaShop#25023]
#### Describe the bug

```
<trans-unit id="03c2e7e41ffc181a4e84080b4710e81e" approved="yes">
        <source>New</source>
        <target state="final">Neuf</target>
        <note>Line: 85</note>
 </trans-unit>
```

The term New product is badly translated in french by "Neuf". It's wrong ! It should be "Nouveau" or "Nouveauté"

Some customers believe only theses products are "new" ( in term of usage state ), it's wrong ! 

#### Expected behavior

New -> Nouveau

#### Steps to Reproduce

1. Create french prestashop
2. Create a product
3. set in settings number of day to have the products considered as new: 30
4. Go website, you will see that translation problem

**Screenshots**

If applicable, add screenshots or screen recordings to help explain your problem.
![image](https://user-images.githubusercontent.com/4645967/122547288-31f3c900-d030-11eb-9146-de0796c4876a.png)

#### Additional information

* PrestaShop version: 1.7.7.4
* PHP version: 7.4

**Comment by florine2623** [comment]
Hello @henri9813 ,

We are aware of this issue, it is already in our debug backlog. This issue is a duplicate of #17455.
It will be implemented in PS 1.7.8.0.
To be informed when it's fixed, please subscribe to the issue mentioned above.

Thanks!

**Comment by florine2623** [comment]
Duplicate of #17455
